src/Persistence/MassTransit.Azure.Table/Configuration/Audit/AzureTableAuditStoreConfiguratorExtensions.cs

```diff
@@ -0,0 +1,127 @@
+namespace MassTransit.Azure.Table
```

Comment:
IBusFactoryConfigurator extension methods should be in the root MassTransit namespace.

Reply:
Sorted